src/Symfony/Component/Form/ChoiceList/ArrayChoiceList.php

```diff
@@ -233,13 +233,13 @@ private function castableToString(array $choices, array &$cache = array())
                 }
 
                 continue;
-            } elseif (!is_scalar($choice)) {
+            } elseif (!is_scalar($choice) && null !== $choice) {
```

Comment:
@Tobion a solution for the placeholder issue is to revert the change on this line so it keeps the old behaviour when `null` is a value option. WDUT ?